index.html

```diff
@@ -2,17 +2,17 @@
 <html>
     <head>
         <title>Angular 2 Quickstart</title>
-        <link rel="stylesheet" href="src/style/style.css">
-        <script src="jspm_packages/system-polyfills.js" defer></script>
-        <script src="jspm_packages/system.js" defer></script>
+        <link rel="stylesheet" href="dist/style/style.css" />
+        <script src="node_modules/systemjs/dist/system-polyfills.js" defer></script>
+        <script src="node_modules/systemjs/dist/system.js" defer></script>
         <script src="config.js" defer></script>
         <script src="node_modules/angular2/bundles/angular2.dev.js" defer></script>
```

Comment:
To tutaj jest chyba niepotrzebne. Jak są importy w bootstrapie i App, to powinno wystarczyć.